src/main/java/uk/gov/justice/probation/courtcaseservice/service/OffenderService.java

```diff
@@ -107,8 +109,10 @@ public ProbationRecord getProbationRecord(String crn, boolean applyDocumentFilte
                 probationRecord.setAssessment(findMostRecentByStatus(assessments).orElse(null));
             });
         } catch (OffenderNotFoundException e) {
+            telemetryService.trackTelemetryEvent(TelemetryEventType.GRACEFUL_DEGRADE);
```

Comment:
It would be good to include the specifics of what cause the degradation, could we send the info message as well? It's already logged obviously but saves us the hassle of having to correlate the two... in fact we could probably get rid of the log.warn() as this kinda supersedes it 🤔

Reply:
It's also possible that in future we'll have different degradation events so will need a way to differentiate

Reply:
In fact might be worth breaking it down a bit for easy querying, something along these lines:
```
{
  "description": "Excluding offenderAssessments", 
  "cause": exception,
  "crn": crn
}
```

Reply:
Given there is only one at the moment, it could only be for offender assessments. 
I followed the ticket and the maxim "do the simplest thing that works". I'll do it but stuff like this should be on the ticket at the outset.

Reply:
We already have an alert for the offender assessments failure so why do we have this telemetry ? I am a bit confused by the intended approach TBH

Reply:
Apologies, you're right I didn't make it clear on the ticket, will try to be more explicit in future.